### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/163ad9af0c7d5a15a8016f281c134993f7a12019

### CODE REVIEW

File: en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+developer-mdnplus-notifications = Notifications
+developer-mdnplus-development-in-real-time = Development in real time: Get custom alerts
+developer-mdnplus-the-web-doesnt-have-a = The Web doesn’t have a changelog, but { -brand-name-mdn } can help. Follow pages and get customizable notifications when documentation changes, CSS features launch, and APIs ship.
+developer-mdnplus-collections = Collections
+developer-mdnplus-build-your-perfect-library = Build your perfect library. Or let us build it for you.
+developer-mdnplus-no-more-haphazard-hunting = No more haphazard hunting through the vast virtual library: unleash your inner curator and collect your favorite articles in one place for convenient consultation.
+developer-mdnplus-mdn-offline = { -brand-name-mdn } offline
+developer-mdnplus-mdns-entire-library-at-your = { -brand-name-mdn }’s entire library at your fingertips: offline
+developer-mdnplus-taking-your-projects-beyond = Taking your projects beyond the nearest wifi signal? Say goodbye to inaccessible pages or cluttered tabs. With { -brand-name-mdn-plus }, have the fully navigable resources of { -brand-name-mdn } at your disposal even when offline.
```
Comment (peiying2) [inline en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl:28]: Should `wifi` be `WiFi`?

**Comment by peiying2** [comment]
@craigcook let me know when this is ready for review.

File: en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl (hunk 1 above)
Comment (craigcook) [reply en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl:28]: I'm not sure what's official but if we most often use WiFi elsewhere I'd assume that's the standard. I'll update.

Comment (peiying2) [reply en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl:28]: I looked up else where, the recommendation is Wi-Fi, ^-^

**Pull request closed without merging**